test/data/Alerts/alerts-load_example_data.py

```diff
@@ -11,21 +11,20 @@
 import logging
 import json
 from contextlib import contextmanager
-from optparse import OptionParser, TitledHelpFormatter
+from argparse import ArgumentParser
 
 from WMCore.Database.CMSCouch import CouchServer, Database, Document
 
 
 def _processCmdLineArgs(args):
     usage = \
 """usage: %prog options"""
-    form = TitledHelpFormatter(width = 78)
```

Comment:
Note I didn't find an `argparse` equivalent of `TitledHelpFormatter`. Do we care about this?